docs/book/installation.rst

```diff
@@ -61,7 +61,7 @@ And now you can use gulp for installing views, by just running a simple command:
 
 .. code-block:: bash
 
-    $ npm run gulp
+    $ gulp
```

Comment:
We should still use `npm run gulp`, because `npm install` installs it locally, but not globally.